### PR TITLE
Handling IPC message from Player to terminate process

### PR DESCRIPTION
**Pull request by stulees**

### CODE REVIEW

**Comment by stulees** [comment]
@rodrigopavezi I wasn't sure if it was possible since it would require the integration test to fork rise cache process. Since the code change in this PR is only leveraging node API (child_process module), I chose not to spend any time trying. Do you think it's possible and worth trying?

**Comment by rodrigopavezi** [comment]
@stulees got it, should be fine. looks good 👍  cheers

File: app/app.js [hunk 1]
```diff
@@ -33,6 +33,13 @@ const AppFactory = function() {
         logger.error("Uncaught exception", err.message);
       });
 
+      process.on( "message", ( msg ) => {
+        if ( msg && typeof msg === "string" && msg === "quit" ) {
+          logger.info("Rise Cache is terminating");
```
Comment (tejohnso) [inline app/app.js]: I think we should be more specific for the sake of clarity:
`Quit process message received. Rise Cache will terminate.`

**Comment by stulees** [comment]
@tejohnso Message changed, thanks

**Pull request merged**